README.md

```diff
@@ -132,7 +132,42 @@ Example: Train.jsonl
 
 ## Supported Models
 
-Current supported and tested models are `Llama3` (8B configuration has been tested) and `GPTBigCode`.
+Current supported and tested models are `Granite v3`, `Llama3` (8B configuration has been tested) and `GPTBigCode`.
```

Comment:
```suggestion
Current supported and tested models are `Granite 3.0`, `Llama3` (8B configuration has been tested) and `GPTBigCode`.
```

Reply:
The above is also not fully accurate as it leaves out the mistral, mixtral, and granite with llama models (small code models) which wouldn't be distinguished with llama3 or granite3 or gptbigcode.